java/common/localstack-test/src/main/java/sleeper/localstack/test/LocalStackTestBase.java

```diff
@@ -87,4 +88,11 @@ public static String createSqsQueueGetUrl() {
         return SQS_CLIENT.createQueue(UUID.randomUUID().toString()).getQueueUrl();
     }
 
+    public static String createSqsQueueGetUrlV2() {
+        return SQS_CLIENT_V2.createQueue(software.amazon.awssdk.services.sqs.model.CreateQueueRequest.builder()
+                .queueName(UUID.randomUUID().toString())
+                .build())
+                .queueUrl();
+    }
```

Comment:
It seems like it just makes extra work if we have two versions of this helper. We can either use the existing one as it is, or migrate it to AWS SDK v2 now.